pxr/usd/usdGeom/bboxCache.cpp

```diff
@@ -439,7 +465,7 @@ UsdGeomBBoxCache::_ComputeBoundWithOverridesHelper(
     }
 
     GfBBox3d result;
-    UsdPrimRange range(prim);
+    UsdPrimRange range(prim, _primPredicate);
```

Comment:
I was a bit unsure about this change. Looking at where this is called, it seemed to me that this should be following the same predicate as the other traversals, but it was not before. Let me know if I need to roll this back